modules/Chromecast/resources/sender/chromecast.js

```diff
@@ -42,7 +42,9 @@
                     this.isInIframeApi = false;
                 } catch ( e ) {
                     window[ '__onGCastApiAvailable' ] = this.toggleCastButton.bind( this );
-                    kWidget.appendScriptUrl( this.CAST_SENDER_V3_URL );
+                    // PSVAMB-4560
+                    // For some reason, Chrome appends script to a player's iframe parent document unless specified explicitly
+                    kWidget.appendScriptUrl( this.CAST_SENDER_V3_URL, null, window.document );
```

Comment:
@vladm3 calling the method without this does the same:

https://github.com/kaltura/mwEmbed/blob/705544054bd4d2ae7dda67c6b86a9fee0d263641/kWidget/kWidget.js#L2339-L2342